### DISCUSSION TITLE
Missing button in the Advanced Logbook View: With selected: Not Required

### DISCUSSION THREAD

**Discussion by DJLax5** [magicbug/Cloudlog#2051] [Ideas / Feature Requests]
I would much apprecheate a button in the Advanced Logbook View which marks the selected QSOs as QSL Not required!

**Comment by AndreasK79** [comment]
I added a button to do this now. It's available in the dev branch if you use that, it not need to wait till Peter issues a new release.

**Reply by DJLax5** [reply]
Amazing, thank you so much!!!!
